eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.2.10</VersionPrefix>
+    <VersionPrefix>17.4.5</VersionPrefix>
```

Comment:
This is a weird diff - vs17.4 (target of this OR) has 17.4.4

https://github.com/dotnet/msbuild/blob/8e9cb295288f13095e9b7739cc4cbb04e9d246d2/eng/Versions.props#L5